internal/common/vote/vote.go

```diff
@@ -0,0 +1,191 @@
+package vote
+
+import (
+	"math/rand"
+	"time"
+)
+
+func Get_votes_for_rule(ruleID string, islandID int) map[int][]int {
```

Comment:
What does this function do? Can you document it please?

Reply:
If this function is supposed to reach out to the island, it should return `map[shared.ClientID][]int`

Reply:
It is expected to get voting information from each island, but it seems that how will each island vote under certain circumstances has not been determined by design team, so I directly generate random votes inside the function instead.

Reply:
Right, thanks :) -- this should be in the comment of the function then, with a `TODO` noting that it should be changed when the island voting interface is added.

Can you also ping the voting design rep to confirm this? I assume we need voting for the islands in the MVP. I'm not sure whose responsibility it is to add it to the client interface.